task2/list.c

```diff
@@ -0,0 +1,151 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "list.h"
+
+typedef struct Node
+{
+    Value value;
+    struct Node *next;
+} Node;
+
+typedef struct List
+{
+    Node *head;
+} List;
```

Comment:
```suggestion
} SortedList;
```
Название типа должно быть более информативным. Разве абстрактный тип, который мы предоставляем пользователю, это просто список?